src/makeCalculator.js

```diff
@@ -4,7 +4,68 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(value) {
+      this.result += value;
+
+      return this;
+    },
+
+    subtract(value) {
+      this.result -= value;
+
+      return this;
+    },
+
+    multiply(value) {
+      this.result *= value;
+
+      return this;
+    },
+
+    divide(value) {
+      if (value !== 0) {
+        this.result /= value;
+      } else {
+        throw new Error("Can't divide by 0!");
+      }
+
+      return this;
+    },
+
+    operate(operation, value) {
+      if (value === undefined) {
```

Comment:
There is no need to check if the value is `undefined`. If no value is provided, JavaScript will automatically set it to `undefined`, and the operation functions should handle this case correctly. This is a code quality issue.